scripts/release/make-release.py

```diff
@@ -97,6 +89,10 @@ def fetch():
         write_release_log(release_message)
         commit_tag_and_push()
 
-        print("Done!")
-        print("""When you come to deploy this release, the RELEASE_LOG.md file
+        print("""Done"
+No changes have been pushed, so please review and then push using 
+
+git push --tags
```

Comment:
Does this not push _just_ the tags?  Apparently you can do `git push --follow-tags` and it'll push the tags referenced on pushed commits (https://stackoverflow.com/questions/3745135/push-git-commits-tags-simultaneously)